.github/workflows/build.yaml

```diff
@@ -7,7 +7,7 @@ on:
       - develop
       - release*
   pull_request:
-      type: [opened, reopened, edited]
+      type: [opened, reopened, edited, synchronize]
```

Comment:
:)
Maybe delete the pull_request type?

Reply:
The changes in the repository not always links to the changes in branches of this repository. For example if PR created from repository fork - you don't have any changes in the repo branches